README.md

```diff
@@ -170,12 +171,21 @@ Immediately triggers a new search for the given text
 
 - text `String` A word of sequence of words to search for
 
-#### `search.playResult(result)`
+**Returns:**
+- retVal `Promise` - A promise that will resolve with the results of the search you just performed.
+The promise will be rejected with a timeout error if the search takes too long.
+
+#### `search.playResult(searchText, result)`
 Immediately attempts to play the given result.  If we fail to play the given result
 an error will be thrown.
 
+- searchText `String` The search text that causes the search result to appear
```

Comment:
Well this might just be a personal implementation thing, but I call the `gmusic` and `gmusic-ui` methods remotely with strings for `namespace`, `method` and an array of arguments.

Sequentially calling two methods with a promise doesn't work very well when you are calling remotely over basically a JSON only interface.  That's why I like the idea of having each required action (in this case playing a song from a known set of search results) to be a single method call.

Similar to how our playlists functionality has a `playWithTrack()` method that navigates to the playlist and plays a given track in said playlist.  Why make what is at heart a 1 step operation, a 2 step one.